src/scrabble.js

```diff
@@ -1,6 +1,35 @@
+const ScoreUtil = require('./utils/Scores')
+
```

Comment:
Not many pushes (I did the same), think we should probably push at every stage more.